Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```diff
@@ -3621,6 +3621,180 @@ script:
     description: Remove a tag from one or more endpoints.
     execution: false
     name: core-remove-endpoint-tag
+  - name: core-list-users
+    description: Retrieve a list of the current users in your environment.
+    arguments: []
+    outputs:
+    - contextPath: Core.User.user_email
+      description: Email address of the user
+      type: string
+    - contextPath: Core.User.user_first_name
+      description: First name of the user
+      type: string
+    - contextPath: Core.User.user_last_name
+      description: Last name of the user.
+      type: string
+    - contextPath: Core.User.role_name
+      description: Role name associated with the user.
+      type: string
+    - contextPath: Core.User.last_logged_in
+      description: Timestamp of when the user last logged in.
+      type: integer
+    - contextPath: Core.User.user_type
+      description: Type of user.
+      type: string
+    - contextPath: Core.User.groups
+      description: Name of user groups associated with the user, if applicable.
+      type: array
+    - contextPath: Core.User.scope
+      description: Name of scope associated with the user, if applicable.
+      type: array
+  - arguments:
+    - description: |
+        Unique ID of a specific user.
+        User ID could be in the following format: dummy/dummy
+      name: user_id
+    - description: Limit the number of users that will appear in the list. By default, the limit is 50 users.(use limit when you don't asking for specific user)
```

Comment:
is this the maximal number of users that _appear_, or the maximal number of users to query? There's a small difference.